### DISCUSSION TITLE
New Talk!

### DISCUSSION THREAD

**Discussion by MGomezN** [fatiando/community#153] [Announcements]
![thumbnai](https://github.com/fatiando/community/assets/29388479/01509e40-9f36-4956-b159-027c43debb0f)
We recommend watching this talk by @santisoler . You will find a clear, didactic explanation and live coding demonstrations of the available open-source tools for analyzing this data type.

Watch the talk: https://www.youtube.com/watch?v=WFgbd56-xLg
  
Link to the slides and notebooks:  https://github.com/santisoler/2024-uh

**Comment by santisoler** [comment]
Thanks @MGomezN for uploading the talk to YT! Much appreciated!
